Sources/Sentry/SentryHub.m

```diff
@@ -298,15 +299,15 @@ - (SentryId *)captureEvent:(SentryEvent *)event
     return SentryId.empty;
 }
 
-- (id<SentrySpan>)startTransactionWithName:(NSString *)name operation:(NSString *)operation
```

Comment:
These should probably actually return a SentrySpan. SentryTracer should remain as an implementation detail of the SDK, not exposed to the public. But the ownership semantics will need to be reworked, because since we technically did return the tracer instance, and just "called it" a SentrySpan, meant that it wasn't deallocated (unless it was kept alive by the reference SentrySpan.tracer, not sure about that).